### PR TITLE
Gui timeout cb

### PR DESCRIPTION
**Pull request by gamecoder95**
Added GUI callback to display a response after resending a request 3 times.

### CODE REVIEW

File: auctionsystem/client.py [hunk 1]
```diff
@@ -184,22 +186,22 @@ def rcv_highest(self, item_num, amount):
...
             print("You are the winner of item {}, bought for {}!".format(item_num, amount))
         self.bidding_ended(item_num)
 
     def rcv_bid_over(self, item_num, amount):
         # Item has been sold to another client
         self.bidding_ended(item_num)
-        if self.gui_cb:
-            self.gui_cb(MESSAGE.BID_OVER, item_num, amount)
+        if self.gui_update_cb:
+            self.gui_update_cb(MESSAGE.WIN, item_num, amount)
```
Comment (samratdebroy) [inline auctionsystem/client.py]: Shouldn't this be MESSAGE.BID_OVER?

**Review by samratdebroy — CHANGES_REQUESTED** [review]
just need to confirm is you wanted WIN instead of BID_OVER

File: auctionsystem/client.py (hunk 1 above)
Comment (gamecoder95) [reply auctionsystem/client.py]: Yes it is...that may have been due to the merge conflict.

Comment (gamecoder95) [reply auctionsystem/client.py]: Fixed

**Review by samratdebroy — APPROVED** [review]

**Pull request merged**